tests/index.html

```diff
@@ -40,6 +40,15 @@
       (function() {
         window.EmberENV = window.EmberENV || {};
 
+        // Determine edition, and apply correct EmberENV settings
+        var edition = QUnit.urlParams.edition || 'octane';
+
+        if (edition === 'octane') {
+          EmberENV._TEMPLATE_ONLY_GLIMMER_COMPONENTS = true;
```

Comment:
is there a source of truth for this? not a big deal if not just curious

Reply:
https://github.com/ember-cli/ember-cli/blob/master/blueprints/app/files/config/optional-features.json